src/Routes/ImageManager/ImageSetsTable.js

```diff
@@ -91,12 +106,14 @@ const createRows = (data, baseURL, history, navigate) => {
 
       image_set?.Version,
       {
-        title: image_set?.UpdatedAt ? (
-          <DateFormat date={image_set?.UpdatedAt} />
-        ) : (
-          'Unknown'
-        ),
+        title: distributionMapper[image_set?.Distribution],
       },
+      image_set?.OutputTypes[0] == 'rhel-edge-installer' &&
+      image_set?.OutputTypes[1] == 'rhel-edge-commit'
+        ? 'Bare Metal Installer'
+        : image_set?.OutputTypes[0] == 'rhel-edge-commit'
+        ? 'Update Only'
+        : null,
```

Comment:
in my local database the OutputTypes looks like {"rhel-edge-commit","rhel-edge-installer"} so the commit is first,
maybe we should simply check if image_set?.OutputTypes.length == 2 ? 'Bare Metal Installer' :  'Update Only'
what do you think ?

Reply:
good idea

Reply:
or if you need to be more explicit: 

image_set?.OutputTypes.includes('rhel-edge-installer') it's 'Bare Metal Installer' else if image_set?.OutputTypes.includes('rhel-edge-commit') then 
'Update Only' else 'unknown'